src/utils/roleDefinition.ts

```diff
@@ -1,28 +1,58 @@
-import { RoleDefinition } from '@microsoft/microsoft-graph-types';
+import { UnifiedRoleDefinition } from '@microsoft/microsoft-graph-types';
 import { cli } from '../cli/cli.js';
 import { formatting } from './formatting.js';
 import { odata } from './odata.js';
+import request, { CliRequestOptions } from '../request.js';
 
 export const roleDefinition = {
   /**
-   * Get a directory (Microsoft Entra) role
+   * Get an Entra ID (directory) role by it's name
```

Comment:
I've made some comments about this file on #6487 that this PR should be based on as well.

Reply:
fixed